### PR TITLE
docs: add some missing imports and update some property names in code examples in pipelines README

### PR DESCRIPTION
**Pull request by ghost**
Ran across some of these when preparing the How-To topic for the CDK Dev Guide; backporting to the README.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/pipelines/README.md [hunk 1]
```diff
@@ -100,45 +104,45 @@ to build. The Pipeline will be provisioned in account `111111111111` and region
...
-        oauthToken: SecretValue.secretsManager('GITHUB_TOKEN_NAME'),
-        trigger: codepipeline_actions.GitHubTrigger.POLL,
-        // Replace these with your actual GitHub project name
-        owner: 'OWNER',
-        repo: 'REPO',
-      }),
-
-      synthAction: SimpleSynthAction.standardNpmSynth({
-        sourceArtifact,
+    constructor(scope: Construct, id: string, props?: StackProps) {
```
Comment (shivlaks) [inline packages/@aws-cdk/pipelines/README.md:107]: we use 2 spaces in our code-base (for TS), so I think we shouldn't change the spacing here

File: packages/@aws-cdk/pipelines/README.md [hunk 2]
```diff
@@ -27,7 +27,11 @@ same, the *CDK Pipelines* library takes care of the details.
 will work, see the section **CDK Environment Bootstrapping** below).
 
 ```ts
-import { Construct, Stage } from '@aws-cdk/core';
+import { DatabaseStack, ComputeStack } from '../lib/my-stacks';
```
Comment (shivlaks) [inline packages/@aws-cdk/pipelines/README.md:30]: do we need to provide a couple sentences to introduce these stacks and what they contain?
Adding the import provides more of a clue of how the app is structured. However, it does not quite bridge the gap of understanding what they are / what they contain.

File: packages/@aws-cdk/pipelines/README.md (hunk 1 above)
Comment (Jerry-AWS) [reply packages/@aws-cdk/pipelines/README.md:107]: I'm gonna blame VS Code for that... yeah

File: packages/@aws-cdk/pipelines/README.md (hunk 2 above)
Comment (Jerry-AWS) [reply packages/@aws-cdk/pipelines/README.md:35]: I've added some comments.

**Comment by Jerry-AWS** [comment]
@shivlaks Have another look

**Pull request merged**